pkgs/top-level/python-packages.nix

```diff
@@ -8587,6 +8587,10 @@ self: super: with self; {
 
   pyalgotrade = callPackage ../development/python-modules/pyalgotrade { };
 
+  pyalmond = callPackage ../development/python-modules/pyalmond { };
```

Comment:
from ofborg: 

    pkgs/development/python-modules/pyalmond: No such file or directory

Reply:
Whoops, bad merge. Should be fixed now.